opendbc/safety/safety/safety_toyota.h

```diff
@@ -224,6 +227,18 @@ static bool toyota_tx_hook(const CANPacket_t *to_send) {
       }
     }
 
+    if (addr == 0x183) {
+      int desired_accel = (GET_BYTE(to_send, 0) << 8) | GET_BYTE(to_send, 1);
+      desired_accel = to_signed(desired_accel, 16);
+
+      bool violation = !toyota_secoc;  // Only SecOC cars may transmit this message
```

Comment:
this should already be handled by the TX msgs

Reply:
Roger. Will remove